python/modules/qcdSmearProducer.py

```diff
@@ -1,95 +0,0 @@
-#!/usr/bin/env python
```

Comment:
So the qcdsmearproducer will be deleted? Where do you host the code?

Reply:
The QCD smearing code is stored in my QCDSmearing branch which I did a PR to the susy15/QCDSmearing branch. You need to look at that PR also. It is very far behind dev_v3 so I could automatically merge it. I also do not have permissions to update it.